.github/workflows/functions.yaml

```
@@ -0,0 +1,32 @@
+name: Functions deploy test
+
+on: workflow_dispatch
+
+concurrency:
+  group: functions-deploy-${{ github.ref }}
+
+env:
+  CI: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

IIRC, this has to be in quotes (for reasons I don't fully understand)

```suggestion
          node-version: "16"
```